core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1792,12 +1792,36 @@ class SparkContext(config: SparkConf) extends Logging {
       }
     }
 
+    def addRemoteJarFile(path: String): String = {
```

Comment:
Better to change to `checkRemoteJarFile`, here in this method it only checks the jar file.

Reply:
origin addFileJar will also check jar exists. all same to local jar file .

Reply:
`addJarFile` also adds the jar file to fileserver, that's the key purpose there, not just checking.

Reply:
Reasonable, done .